### ISSUE TITLE
extra relaxing

### ISSUE THREAD

**Issue by wdduncan** [ontodev/robot#463]
I like the relax method (thanks!). But, is it possible to add functionality so that classes with nested subclasses can be further decomposed? My use case is SNOMED. The image shows the axioms for 'Malignant neoplastic disease (disorder)'. 
It would be helpful if this class could be further decompose the 'Role group' axioms.

<img width="1099" alt="snomed malignant neoplasm" src="https://user-images.githubusercontent.com/3186638/56509512-2ca09f80-64f5-11e9-8f75-52d369a3efab.png">

**Comment by balhoff** [comment]
@wdduncan I have a tool that outputs all inferable direct existential relations as triples, using a reasoner: https://github.com/NCI-Thesaurus/ncit-utils It isn't really documented but I can help you use it. This would be an alternative approach, compared to what I would call a "structural" transformation provided by `relax`. ROBOT also has the [materialize](http://robot.obolibrary.org/materialize) command which may actually do exactly what you want, if you want to see them as standard existential restrictions. This is kind of similar to my tool, but maybe less scalable for many relations at once.

**Comment by cmungall** [comment]
Are we talking about slightly different things?

I think Bill wants a structural transform that writes subClassOf from elements of intersections in equivalence axioms; currently this is hardcoded to one level of nesting? Is that correct Bill?

The other is an inference operation that materializes existentials up to one level -- which will likely require additional axioms to reduce the level of nesting (e.g. property chains)?

Also Bill you may be interested in https://github.com/hsolbrig/SNOMEDToOWL it does a lot of simplification for you cc @hsolbrig

**Comment by wdduncan** [comment]
@cmungall yes, that is correct. To use materialize, wouldn't I have to specify all the relationships I wanted to compose? I am not sure ...

Thanks for the pointer to SNOMEDToOWL!
